docs/production/upgrade-or-modify.md

```diff
@@ -249,7 +249,8 @@ working correctly.
 ### Upgrading from Ubuntu 16.04 Xenial to 18.04 Bionic
 
 1. Upgrade your server to the latest Zulip `2.1.x` release, since
-   newer releases don't support Ubuntu 16.04 Xenial.
+   newer releases don't support Ubuntu 16.04 Xenial.  The instructions
+   below will not work if you are running Zulip 3.0.
```

Comment:
I think we want to express here that Zulip 3.0 does not support Ubuntu Xenial and so you must upgrade to 2.1.x and do this process first; the text here presupposes you can be running Zulip 3.0 on Xenial (which is not possible).

Reply:
I'll just tweak this before merging, since the rest looks great.